io/net/ethtool/ethtool_tests.sh.data/README.txt

```diff
@@ -0,0 +1,14 @@
+Tests the network driver and interface with 'ethtool' command.
+12 different options of ethtool command are tested for each
+of the interfaces specified in 'config' file
```

Comment:
If you' re going to use the multiplex file for the devices, remember to drop this line.